Formula/fluxctl.rb

```diff
@@ -0,0 +1,33 @@
+class Fluxctl < Formula
+  desc "fluxctl talks to Weave Flux and helps you deploy your code"
+  homepage "https://github.com/weaveworks/flux"
+  url "https://github.com/weaveworks/flux.git",
+      :tag => "1.7.1",
+      :revision => "185dfdf5a317d8ed2042f6afe2d6b94950282aff"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    arch = MacOS.prefer_64_bit? ? "amd64" : "x86"
+    dir = buildpath/"src/github.com/weaveworks/flux"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+
+    cd dir do
+      system "dep", "ensure"
```

Comment:
This is different from most of our `dep` formulas, which use `dep ensure -vendor-only`, but not `dep prune`. Can you explain?

Reply:
Apparently I hadn't seen [recent change](https://github.com/weaveworks/flux/commit/b1c422f681f99b1c3319b42673b8b807525866d4#diff-60245d3d9a7e8104a01f0c4d564dfd7b) in the build instructions yet. Will remove `dep prune` as it's not necessary any more.